src/pages/Personas/Personas.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useState } from 'react';
+import styled from 'styled-components';
+import { TopNav } from 'components/TopNav/TopNav';
+import { PageWrapperSpaceBetween } from 'components/PageWrapper';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { GET_PERSONAS, GetPersonaType, SET_DEFAULT_PERSONA } from 'global/graphqls/Persona';
+import { PersonaCard } from 'components/PersonaCard/PersonaCard';
+import { Button } from 'components/Button';
+import Carousel from 'components/Carousel/Carousel';
+import { useUserContext } from 'global/UserContext/UserContext';
+import { gqlPersona } from 'global/graphqls/schema';
+
+const StyledButton = styled(Button)`
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const DefaultBlock = styled.div`
```

Comment:
Just for the notice, we have layout container system I added previously in `FlexBox`. You can check it.